src/nvim/options.lua

```diff
@@ -188,7 +188,7 @@ return {
     },
     {
       full_name='belloff', abbreviation='bo',
-      deny_duplicates=true,
+      deny_duplicates=true, -- luacheck: ignore 314
       type='string', list='comma', scope={'global'},
       deny_duplicates=true,
```

Comment:
That's an actual error that should not be ignored.

Reply:
Thanks. Fixed it